mslib/utils/thermolib.py

```diff
@@ -35,7 +35,7 @@
 from metpy.xarray import preprocess_and_wrap
 import metpy.calc as mpcalc
 
-exporter = Exporter(globals())
+exporter = Exporter(locals())
```

Comment:
This doesn't make sense. The Exporter is a metpy helper class that maintains the "__all__" lsit of module symbols.
Also, is there a difference in globals/locals at this point in the code???
I do not now why we use this anyway. I suggest removing it and its decorator use below.

Reply:
good idea :)